packages/client/components/AnalyticsPage.tsx

```diff
@@ -89,39 +89,39 @@ const TIME_TO_RENDER_TREE = 100
 
 const AnalyticsPage = () => {
   const atmosphere = useAtmosphere()
-  const [isGAInitialized, setIsGAInitialized] = useState(false)
+  if (gaMeasurementId) {
+    ReactGA.initialize(gaMeasurementId, {
```

Comment:
There won't be duplicate initializations as it's [protected](https://github.com/PriceRunner/react-ga4/blob/master/src/ga4.js#L183) inside `ReactGA`